examples/bls.rs

```diff
@@ -0,0 +1,8 @@
+use ethereum_consensus::crypto;
+use ethereum_consensus::crypto::SecretKey;
+
+fn main() {
+    let sk = SecretKey::random();
+    let pk = sk.public_key();
+    println!("{:?}", pk);
+}
```

Comment:
and a minor thing but i tend to include trailing newlines -- what do you think?

Reply:
agree